apps/backoffice-v2/src/lib/blocks/variants/ManualReviewBlocks/ManualReviewBlocks.tsx

```diff
@@ -0,0 +1,14 @@
+import { BlocksComponent } from '@/lib/blocks/components/BlocksComponent/BlocksComponent';
+import { useManualReviewBlocksLogic } from '@/lib/blocks/variants/ManualReviewBlocks/hooks/useManualReviewBlocksLogic/useManualReviewBlocksLogic';
+import { NoBlocks } from '@/lib/blocks/components/NoBlocks/NoBlocks';
+
+export const ManualReviewBlocks = () => {
+  const { blocks, isLoading } = useManualReviewBlocksLogic();
+
+  return (
+    <>
+      <BlocksComponent blocks={blocks} />
```

Comment:
Should we show `BlocksComponent` only if `blocks?.length` is truthy?

Reply:
Handled by the package.